rpcs3/Crypto/utils.cpp

```diff
@@ -142,3 +142,98 @@ void mbedtls_zeroize(void *v, size_t n)
 	static void *(*const volatile unop_memset)(void *, int, size_t) = &memset;
 	(void)unop_memset(v, 0, n);
 }
+
+
+// SC passphrase crypto
+
+void get_sc_key_by_revision(int revision, u8* key)
+{
+	switch (revision)
+	{
+	case 2: hex_to_bytes(key, "D413B89663E1FE9F75143D3BB4565274", PASSPHRASE_KEY_LEN * 2); break;
+	case 3: hex_to_bytes(key, "FA72CEEF59B4D2989F111913287F51C7", PASSPHRASE_KEY_LEN * 2); break;

```

Comment:
std::memcpy(key, "\xFA\x72\xCE"...

Reply:
that's not really readable.

Reply:
It has seperators for different bytes which is crucial for making it readable.

Reply:
what i mean is: you can't just copy paste it or search for it. (for example online)

Reply:
Just replace the seperator with whatever you want to search, nearly every text editor supports this feature. (including no seperator by using blank field)
Furthermore you can't do the opposite without seperator to begin with.

Reply:
So you can replace "\x" with ", 0x" for example etc which is c syntax and the syntax of lots of online tools as well.

Reply:
done